index.html

```diff
@@ -64,19 +95,22 @@
     <div class="container">
       <!-- Example row of columns -->
       <div class="row">
+        <h1>
+         Featured Poets
+        </h1>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h2>Alfred Tennyson</h2>
+          <p>Tennyson remains one of the most popular poets of the English Language. His themes range from mythology to the every day. Learn why "'Tis better to have loved and lost / Than never to have loved at all"</p>
+          <p><a class="btn btn-default" href="#" role="button">View Details &raquo;</a></p>
```

Comment:
Perhaps this buttons can have access to additional information down the road. Kind of open it up a little bit more for users.